styles/card2.css

```diff
@@ -0,0 +1,67 @@
+.card2-container {
+  margin: 4rem 1rem;
+
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-around;
+}
+
+.card2 {
+  display: inline-block;
+  max-width: 500px;
+  max-height: 400px;
+  background-color: var(--color-grey-light-5);
+
+  display: flex;
+  justify-content: center;
+  margin: 1rem 0;
+  cursor: pointer;
+}
+
+.card2__cover {
+  max-width: 204px;
```

Comment:
same here, you should use rem, use consistent units throughout

Reply:
fixed, done.